recipes/_configure_server.rb

```diff
@@ -41,13 +41,35 @@
 case node['splunk']['node_type']
 when :search_head, :server
   clusters = CernerSplunk.all_clusters(node).collect do |(cluster, bag)|
-    stanza = "clustermaster:#{cluster}"
     master_uri = bag['master_uri'] || ''
+    deployer_uri = bag['deployer_uri'] || ''
     settings = bag['settings'] || {}
     pass = settings['pass4SymmKey'] || ''
+    shc_replication_factor = settings['_cerner_splunk_shc_rep_factor'] || ''
+
+    if node['splunk']['config']['sh_cluster']
+
+      fail "Deployer URI is not specified for #{cluster}" if deployer_uri.empty?
+
+      replication_ports = bag['replication_ports'] || {}
+      replication_ports.each do |port, port_settings|
+        ssl = port_settings['_cerner_splunk_ssl'] == true
+        stanza = ssl ? "replication_port-ssl://#{port}" : "replication_port://#{port}"
+        server_stanzas[stanza] = port_settings.delete_if do |k, _|
+          k.start_with? '_cerner_splunk'
+        end
+      end
+
+      server_stanzas['shclustering'] = {}
+      server_stanzas['shclustering']['conf_deploy_fetch_url'] = deployer_uri
+      server_stanzas['shclustering']['disabled'] = 0
+      server_stanzas['shclustering']['replication_factor'] = shc_replication_factor if shc_replication_factor
```

Comment:
Fixed in this commit : 1975a126ed110b99398502dafb844362fa5b7feb
> 
2.1.5 :005 > 2.empty?
NoMethodError: undefined method `empty?' for 2:Fixnum
	from (irb):5

Hence went with 'nil'